### PR TITLE
build(cherow): add fuzz scripts to find bugs

### PR DESCRIPTION
**Pull request by Aladdin-ADD**
Signed-off-by: weiran.zsd <weiran.zsd@alibaba-inc.com>

eslump has successfully found lots of little edge cases in Prettier, so it evidently works. 

to use it, just run `yarn fuzz` in the cherow repo. 😄
image

### CODE REVIEW

**Comment by g-plane** [comment]
LGTM

File: packages/cherow/package.json [hunk 1]
```diff
@@ -50,11 +50,13 @@
...
     "test:watch": "nps test.watch",
     "coverage": "nps coverage",
-    "post_coverage": "nps coverage.post"
+    "post_coverage": "nps coverage.post",
+    "fuzz": "eslump ./test/fuzz/cherow.js .fuzz-output/"
   },
   "devDependencies": {
     "clean-assert": "^2.0.0",
     "coveralls": "^3.0.1",
+    "eslump": "https://github.com/Aladdin-ADD/eslump#fe8be58c359c208418987a4d48edd0034d91737b",
```
Comment (Aladdin-ADD) [inline packages/cherow/package.json]: waiting https://github.com/lydell/eslump/pull/8 get merged.

**Comment by KFlash** [comment]
@Aladdin-ADD sure there will be no conflicts when merging this? I remember in the past..

@fkleuver ?

**Comment by Aladdin-ADD** [comment]
sure!

**Comment by KFlash** [comment]
sounds good! I let u fix this.

**Pull request merged**